src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -437,6 +642,12 @@ def create_or_upgrade_app(
 
         sql_executor = get_sql_executor()
         with sql_executor.use_role(self.role):
+            event_sharing = EventSharingHandler(
+                telemetry_definition=model.telemetry,
+                deploy_root=self.project_root / package.deploy_root,
```

Comment:
isn't `package.deploy_root` already `project_root/deploy_root`?

Reply:
Good point - this has changed recently to a path - it used to be a string.